code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -123,4 +128,17 @@
 
 /datum/deepfryer_special/carrotfries
 	input = /obj/item/reagent_containers/food/snacks/grown/carrot/wedges
-	output = /obj/item/reagent_containers/food/snacks/carrotfries
\ No newline at end of file
+	output = /obj/item/reagent_containers/food/snacks/carrotfries
+
+/datum/deepfryer_special/fried_vox
+	input = /obj/item/organ/external
+	output = /obj/item/reagent_containers/food/snacks/fried_vox
+
+/datum/deepfryer_special/fried_vox/validate(var/obj/item/I)
+	if(!istype(I, input))
```

Comment:
Best to call `..()` here instead of `istype(I, input)`. Outcome is the same but it is more maintainable.